tasks/leave_restart_consul.yml

```
@@ -0,0 +1,19 @@
+---
+
+- name: Consul leave
+  delegate_to: "{{ consul_servers }}"
+  ansible.builtin.command: "{{ consul_binary }} leave {% if consul_acl_enable %} -token {{ consul_acl_master_token }} {% endif %} -http-addr {{ consul_addresses_http }}:{{ consul_ports_http }}"
```

```suggestion
  ansible.builtin.command: "{{ consul_binary }} leave {% if consul_acl_enable %} -token {{ consul_acl_master_token }} {% endif %} -http-addr {{ consul_addresses.http }}:{{ consul_ports.http }}"
```

I _think_ that this should fix the CI error, which is `The task includes an option with an undefined variable. The error was: 'consul_addresses_http' is undefined.`. The port might need to be changed as well.